cmd/dmsg-discovery/commands/root.go

```diff
@@ -44,6 +47,7 @@ func init() {
 
 	RootCmd.Flags().StringVarP(&addr, "addr", "a", ":9090", "address to bind to")
 	RootCmd.Flags().StringVar(&redisURL, "redis", store.DefaultURL, "connections string for a redis store")
+	RootCmd.Flags().StringVar(&whitelistKeys, "whitelist-keys", "", "list of whitelisted keys used network monitor derigstration")
```

Comment:
`list of whitelisted keys of network monitor used for deregistration

Reply:
Done